pmm/kubernetes-cluster-staging.groovy

```diff
@@ -146,10 +146,7 @@ pipeline {
                                 set -o errexit
                                 set -o xtrace
                                 sudo yum -y install curl
-                                sudo curl -Lo /usr/local/sbin/minikube https://github.com/kubernetes/minikube/releases/download/v1.16.0/minikube-linux-amd64
-                                sudo chmod +x /usr/local/sbin/minikube
-                                sudo ln -s /usr/local/sbin/minikube /usr/sbin/minikube
-                                alias kubectl='minikube kubectl --'
+                                /srv/pmm-qa/install_k8s_tools.sh --minikube --kubectl --sudo
```

Comment:
`/srv/pmm-qa/pmm-tests/install_k8s_tools.sh` 
should be the path

Reply:
Thanks!